loginUser.php

```diff
@@ -1,13 +1,14 @@
 <?php
 include 'db.php';
+session_start();
 
 ini_set('display_errors', 1);
 ini_set('display_startup_errors', 1);
 error_reporting(E_ALL);
 
 function logger()
 {
-    include 'db.php';
+    global $conn;
```

Comment:
remove this and make this function generic, pass email and password as a parameters.